src/Symfony/Bundle/SecurityBundle/DependencyInjection/MainConfiguration.php

```diff
@@ -65,7 +65,7 @@ public function getConfigTreeBuilder(): TreeBuilder
                 ->end()
                 ->booleanNode('hide_user_not_found')->defaultTrue()->end()
                 ->booleanNode('erase_credentials')->defaultTrue()->end()
-                ->booleanNode('enable_authenticator_manager')->defaultFalse()->info('Enables the new Symfony Security system based on Authenticators, all used authenticators must support this before enabling this.')->end()
+                ->booleanNode('enable_authenticator_manager')->defaultTrue()->info('Enables the new Symfony Security system based on Authenticators, all used authenticators must support this before enabling this.')->end()
```

Comment:
to what should we change the text as this is not longer true in symfony 6?

Reply:
Let's remove the info message